README.md

```diff
@@ -1 +1,107 @@
+- [opentelemetry-instrumentation-kafka-python](https://pypi.org/project/opentelemetry-instrumentation-kafka-python)
+- [opentelemetry-instrumentation-mysql](https://pypi.org/project/opentelemetry-instrumentation-mysql)
+- [opentelemetry-instrumentation-pika](https://pypi.org/project/opentelemetry-instrumentation-pika)
+- [opentelemetry-instrumentation-psycopg2](https://pypi.org/project/opentelemetry-instrumentation-psycopg2)
+- [opentelemetry-instrumentation-pymemcache](https://pypi.org/project/opentelemetry-instrumentation-pymemcache)
+- [opentelemetry-instrumentation-pymongo](https://pypi.org/project/opentelemetry-instrumentation-pymongo)
+- [opentelemetry-instrumentation-pymysql](https://pypi.org/project/opentelemetry-instrumentation-pymysql)
+- [opentelemetry-instrumentation-pyramid](https://pypi.org/project/opentelemetry-instrumentation-pyramid)
+- [opentelemetry-instrumentation-redis](https://pypi.org/project/opentelemetry-instrumentation-redis)
+- [opentelemetry-instrumentation-requests](https://pypi.org/project/opentelemetry-instrumentation-requests)
+- [opentelemetry-instrumentation-sklearn](https://pypi.org/project/opentelemetry-instrumentation-sklearn)
+- [opentelemetry-instrumentation-sqlalchemy](https://pypi.org/project/opentelemetry-instrumentation-sqlalchemy)
+- [opentelemetry-instrumentation-starlette](https://pypi.org/project/opentelemetry-instrumentation-starlette)
+- [opentelemetry-instrumentation-tornado](https://pypi.org/project/opentelemetry-instrumentation-tornado)
+- [opentelemetry-instrumentation-urllib3](https://pypi.org/project/opentelemetry-instrumentation-urllib3)
+
+
+### Included propagators
+
+```
+opentelemetry-distro
+opentelemetry-instrumentation
+opentelemetry-exporter-otlp-proto-http
+opentelemetry-propagator-jaeger
+opentelemetry-propagator-b3
+opentelemetry-propagator-aws-xray
+opentelemetry-propagator-ot-trace
+```
+
+## Application execution
+
+ Execute the following command. The command will also print the opentelemetry config used.
+
+```
+sumologic-opentelemetry-instrument python3 SCRIPT_NAME.py
+```
+
+## Environment variables config
+
+The wrapper command checks for the following basic OpenTelemetry env variables that are required for the collected data to be useful within Sumo Logic:
+
+- `OTEL_PROPAGATORS`
+- `OTEL_TRACES_EXPORTER`
+- `OTEL_SERVICE_NAME`
+- `OTEL_EXPORTER_OTLP_ENDPOINT`
+- `OTEL_RESOURCE_ATTRIBUTES`
+
+### Propagators
+
+By default, all propgators are enabled (`OTEL_PROPAGATORS=tracecontext,baggage,b3,b3multi,jaeger,xray,ottrace`).
+
+### Exporter.
+
+By default, exported is set to OTLP HTTP (`OTEL_TRACES_EXPORTER=otlp_proto_http`)
+
+### Service name
+
+By default, service name will not be set `OTEL_SERVICE_NAME=`. Overwrite the env var with a string value representing service business logic, such as "FinanceServiceCall". This will appear as a tracing service name in Sumo Logic.
```

Comment:
```suggestion
By default, service name will not be set `OTEL_SERVICE_NAME=`.
Overwrite the env var with a string value representing service business logic, such as "FinanceServiceCall".
This will appear as a tracing service name in Sumo Logic.
```